docs/design/windows-design.md

```diff
@@ -210,8 +210,17 @@ Kube-proxy userspace mode is configured to provide NodePort Service function. A
 "HNS Internal NIC" is provided to kube-proxy to configure Service addresses. The OpenFlow entries for the
 NodePort Service traffic on Windows are the same as those on Linux.
 
-The ClusterIP Service function is implemented by Antrea leveraging OVS. Antrea installs OpenFlow entries
-to select the backend endpoint and performs DNAT on the traffic.
+The ClusterIP Service function is implemented by AntreaProxy. Antrea installs routes from ClusterIP Services
+to a virtual IP (169.254.169.252) and a route from the virtual IP to antrea-gw0. There's an ARP responder flow
+for the virtual IP. The benefit of such design rather than routing a ClusterIP Service directly to antrea-gw0
+is that only one ARP responder flow is needed.
+
+For Services (hostNetwork: false), a packet to them will have its destination IP DNATed to endpoint Pod IP.
+For Services (hostNetwork: true), a packet to them will have its destination IP DNATed to peer Node IP. Then
+its source IP is SNATed to a virtual IP (169.254.169.253). Such SNAT is needed because for reply packet, if
+the destination IP is antrea-gw0 IP, then the packet will not enter OVS pipeline after routed to antrea-gw0.
```

Comment:
Question - why the reply packets to antrea-gw0 IP wont enter OVS pipeline? Do you mean some specific cases like the endpoints are on the same Node, or noEncap mode?

Reply:
I don't think it is for specific cases. Just for reply packets, if the destination is antrea-gw0 IP, then when antrea-gw0 receives the packet, it won't forward the packet to OVS pipeline. Do you think it will?